### DISCUSSION TITLE
Emulating MacOS x86_64 on Apple Silicon machines?

### DISCUSSION THREAD

**Discussion by enlivn** [utmapp/UTM#5510] [macOS]
Hello,
Is emulating MacOS x86_64 on Apple Silicon possible? I've tried a few approaches but none of them have worked. In case anyone's wondering, yes, I can use Rosetta natively to run commands meant for x86 but I have a special use case that needs me to have an "isolated" standalone x86_64 machine.

I'm on an Apple Silicon M1 host running MacOS Ventura 13.5.

### Create bootable `iso`
First, I created a bootable `iso` (I show below what I did for Ventura but I also tried Monterey with the same end result):

1. Download MacOS Ventura via the App Store (links can be found in `Use the App Store` section [here](https://support.apple.com/en-us/HT211683))
2. Run the following commands:
```
sudo hdiutil create -o ./Ventura -size 14079m -volname Ventura -layout SPUD -fs HFS+J -type UDTO -attach
sudo /Applications/Install\ macOS\ Ventura.app/Contents/Resources/createinstallmedia --volume /Volumes/Ventura --nointeraction
hdiutil detach -force /Volumes/Install\ macOS\ Ventura
mv ./Ventura ./Ventura.iso
```

### Create VM in UTM
1. Open UTM.
1. Select Emulate
2. Select 'Other'
3. Pick the iSO image as the bootable image
4. Pick `x86_64` architecture
5. Leave RAM and storage at defaults (4 GB and 64 GB respectively)

When I try starting the VM, the boot image doesn't take. It drops me into the UEFI shell instead. If I hit `ESC` and go into the UEFI boot menu, I see the virtual CD-ROM containing the bootable iso but when I hit `Enter` nothing happens (I stay in the UEFI shell). From my limited understanding, this indicates that I've chosen the wrong architecture for the `iso`. But as far as I know the Ventura (or Monterey) ISOs should be agnostic to x86_64 or arm since both are supported.

I've also gone into the settings of the VM and tried selecting various "Systems" in the `System` tab such as `Standard PC (Q35 + ICH9, 2009) (alias of pc-q35-7.2) (q35)`.

**Comment by F1248** [comment]
This tutorial shows how to emulate a Hackintosh:
https://khronokernel.github.io/apple/silicon/2021/01/17/QEMU-AS.html

**Reply by enlivn** [reply]
Thanks! Based on what they say, the VM is pretty much unusable :( oh well.

**Reply by victorwads** [reply]
Anyone could run some macOs Version on UTM???
